src/hotspot/cpu/aarch64/vm_version_aarch64.cpp

```diff
@@ -163,6 +163,13 @@ void VM_Version::initialize() {
     }
   }
 
+  // Cortex A72
+  if (_cpu == CPU_ARM && (_model == 0xd08 || _model2 == 0xd08)) {
+    if (FLAG_IS_DEFAULT(UseSIMDForMemoryOps)) {
+      FLAG_SET_DEFAULT(UseSIMDForMemoryOps, true);
```

Comment:
What about A73? Should this flag be true for it too?

Reply:
Hi Vladimir,
Thank you for reviewing the changes.
Yes, it can be enabled. However I found only HiKey960/970 by Linaro use A73. Other devices are phones.
I can do this if Linaro engineers agree.

Reply:
ARM ecosystem is really strange :)
In this case keep these changes as it is and let Linaro engineers add it later if they want.